FAQ/mews-li-migration.md

```diff
@@ -0,0 +1,42 @@
+# How to migrate booking engine off mews.li
+
+> ⚠️ Support for this deprecated implementation will end on May 1, which may cause your booking engine to stop working.
```

Comment:
Yes! I'd be ruthless in here.

Reply:
I mean - we could do a redirect ( requires backend pull request, since the infra team is not able to do it on the front-door level ).
And it would solve nothing, it would just prolong the problem.

Also I believe this popup modal will sky-rocket the migration https://github.com/MewsSystems/mews-js/pull/13382

Reply:
I agree, "which will cause your booking engine to stop working."

Reply:
I see, fixed